pkgs/development/python-modules/colcon-coveragepy-result/default.nix

```
@@ -0,0 +1,54 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  colcon,
+  coverage,
+  pytest-cov,
+  pytestCheckHook,
+  setuptools,
+  scspell,
+  writableTmpDirAsHomeHook,
+}:
+
+buildPythonPackage rec {
+  pname = "colcon-coveragepy-result";
+  version = "0.0.8";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "colcon";
+    repo = "colcon-coveragepy-result";
+    tag = version;
+    hash = "sha256-+xjrmiWaDPjoRwjgP4Ui6+vuG4Nc4ur8DdC8ddiXAG0=";
+  };
+
+  build-system = [ setuptools ];
+
+  dependencies = [
+    colcon
+    coverage
+  ];
+
+  nativeCheckInputs = [
+    pytest-cov
```

```suggestion
    pytest-cov-stub
```
https://github.com/NixOS/nixpkgs/pull/414551